DOCKER/build.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+set -e
+
+# Get the tag from the version, or try to figure it out.
+if [ -z "$TAG" ]; then
+	TAG=$(awk -F\" '/Version =/ { print $2; exit }' < ../version/version.go)
```

Comment:
Oy. Cant we just assume tendermint is built already and run `tendermint version` ?